component-samples/scripts/keys_gen.sh

```diff
@@ -0,0 +1,247 @@
+        DENY_LINE=`grep -rn "denylist.publickeyhash" $COMP_PATH/rv/config.properties | cut -d ':' -f 1`
+        sed -i "s/allowlist.publickeyhash\=.*/allowlist.publickeyhash\=\\\\/g" $COMP_PATH/rv/config.properties
+        sed -i "$((ALLOW_LINE+1)),$((DENY_LINE-1))d" $COMP_PATH/rv/config.properties
+        sed -i "$((ALLOW_LINE)) a ${HASHES}" $COMP_PATH/rv/config.properties; sed -i "s/,/,\\\\/g" $COMP_PATH/rv/config.properties;
+}
+
+device_pem_files()
+{
+        # Device EC256 and EC384 pem file creation
+        openssl ecparam -name prime256v1 -genkey -noout -out $CERTS_PATH/device_ec256_private.key
+        openssl req -x509 -key $CERTS_PATH/device_ec256_private.key -out $CERTS_PATH/device_ec256_cert.pem -days 3650 -batch
+        openssl ecparam -name secp384r1 -genkey -noout -out $CERTS_PATH/device_ec384_private.key
+        openssl req -x509 -key $CERTS_PATH/device_ec384_private.key -out $CERTS_PATH/device_ec384_cert.pem -days 3650 -batch
+        cat $CERTS_PATH/device_ec256_cert.pem > $COMP_PATH/device/device.pem
+        cat $CERTS_PATH/device_ec256_private.key >> $COMP_PATH/device/device.pem
+        echo "@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@"
+        echo "************ Creation of Device PEM files completes successfully *************"
+        echo "@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@"
+}
+
+start_generation()
+{
+        arr=($*)
+        if [[ $# == 0 ]]; then
+                echo "For help, provide -h to the script"; exit 1;
+        elif [[ $* =~ "-h" ]]; then
+                usage; exit 1;
+        elif [[ $# == 1 ]]; then
+                arr+=('default')
+        fi
+        mkdir -p $CERTS_PATH
+        for option in ${arr[@]:1}; do
+                case $option in
+                        "-m") component_keys "manufacturer";;
+                        "-o") component_keys "owner";;
+                        "-r") component_keys "reseller";;
+                        "-t") tls_keystore;;
+                        "-d") device_pem_files;;
+                        *)
+                                for i in ${components[@]}; do
+                                        if [[ "${components[@]}" =~ "$i" ]]; then
+                                                component_keys $i
+                                        fi
+                                done
+                                tls_keystore
+                                device_pem_files
+                                break;;
+                esac
+        done
+        for i in $CERTS_PATH/*.hash; do if [[ $i =~ ".hash" ]]; then update_rv_hashes; break; fi; done
+}
+
+start_generation $*
+
+# Final status of the whole execution
+if [[ `ls $CERTS_PATH | wc -l` -gt 0 ]]; then
+        echo "@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@"
+        echo "##################### ${bold}KEYSTORE GENERATION IS SUCCESSFUL${normal} ######################"
```

Comment:
Remove the prints with @ and # characters unless that is used by automation to filter some statement. Also, remove ${bold} and ${normal} comments as sometimes they create issues while parsing the generated output.

Reply:
Removed the prints with @, #, ${bold} and ${normal}.